core/windows_and_tabs/window_snap.py

```diff
@@ -272,14 +276,38 @@ def __init__(self, left, top, right, bottom):
     "fullscreen": RelativeScreenPos(0, 0, 1, 1),
 }
 
+_split_positions = {
+    "split": [
+        _snap_positions["left third"],
+        _snap_positions["center third"],
+        _snap_positions["right third"],
+    ],
+    "clock": [
+        _snap_positions["left"],
+        _snap_positions["top right"],
+        _snap_positions["bottom right"],
+    ],
+    "counterclock": [
+        _snap_positions["right"],
+        _snap_positions["top left"],
+        _snap_positions["bottom left"],
+    ],
+}
+
 
 @mod.capture(rule="{user.window_snap_positions}")
 def window_snap_position(m) -> RelativeScreenPos:
     return _snap_positions[m.window_snap_positions]
 
 
+@mod.capture(rule="{user.window_split_positions}")
+def window_split_position(m) -> list[RelativeScreenPos]:
+    return _split_positions[m.window_split_positions]
+
+
```

Comment:
I don't actually think you need a capture here, the "I want a dictionary so the spoken form is a certain string but the action gets the object that corresponds to that string" mapping can be done with a simple list.

(There are lots of places in community that seem to define unnecessary captures that just wrap a list, for reasons I'm not sure about.)

But I'll follow up to make sure. Feel free to hold off until I've checked 😅

Reply:
lists are a string->string mapping, they can't map to objects

Reply:
derp, right.